<launch>

  <node name="image_publisher"
        pkg="jsk_perception" type="image_publisher.py">
    <rosparam subst_value="true">
      file_name: $(find jsk_perception)/sample/object_detection_example_1.jpg
      encoding: bgr8
      publish_info: false
    </rosparam>
  </node>

  <node name="selective_search"
        pkg="jsk_perception" type="selective_search.py">
    <remap from="~input" to="image_publisher/output" />
  </node>

  <node name="draw_rect_array"
        pkg="jsk_perception" type="draw_rect_array.py">
    <remap from="~input" to="image_publisher/output" />
    <remap from="~input/rect_array" to="selective_search/output" />
  </node>

  <node name="image_view"
        pkg="image_view" type="image_view">
    <remap from="image" to="draw_rect_array/output" />
  </node>

</launch>